.prettierrc

```diff
@@ -0,0 +1,6 @@
+printWidth: 120
```

Comment:
we can leave line width as it is now (190?)

Reply:
https://github.com/FlowCrypt/flowcrypt-browser/pull/4857#discussion_r1060167417